documentation/tutorials/check.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: documentation
+menu: check
+subtitle: 'Tutorial: Using Check with SQL Server'
+---
+# Tutorial: Using Flyway Check with SQL Server
+
+{% include enterprise.html %}
+
+## Introduction
+
+This is an enteprise feature, but the beta version is available to Teams users.
+
+The new `check` command is run prior to migrating, to give us confidence in the state of the post-migration schema by producing a report. 
```

Comment:
...gives **you** confidence...
Want to say something about the report 
...producing a report ... that allows you to better understand the consequences of your planned migrations.